src/engraving/rw/write/twrite.cpp

```diff
@@ -1643,6 +1646,47 @@ void TWrite::write(const Hairpin* item, XmlWriter& xml, WriteContext& ctx)
     xml.endElement();
 }
 
+void TWrite::write(const HammerOnPullOff* item, XmlWriter& xml, WriteContext& ctx)
+{
+    if (item->broken()) {
+        return;
+    }
+    if (!ctx.canWrite(item)) {
+        return;
+    }
+
+    xml.startElement(item);
+    if (ctx.clipboardmode()) {
+        xml.tag("stemArr", Slur::calcStemArrangement(item->startElement(), item->endElement()));
```

Comment:
This can be removed now we've merged https://github.com/musescore/MuseScore/pull/28152